wolfssl/wolfcrypt/ecc.h

```diff
@@ -77,6 +77,15 @@
     #include <wolfssl/wolfcrypt/sp_int.h>
 #endif
 
+#if defined(MAX3266X_MATH)
```

Comment:
Consider these being in wolf_math.h. These are not ecc specific. I recommend consolidating the mapping for these low level math functions into wolf_math.h